test/ExecutionFramework.h

```diff
@@ -241,6 +241,11 @@ class ExecutionFramework
 		return result;
 	}
 
+	bytes const& bytecode() const
```

Comment:
I think this is misleading, it is not the bytecode but the last response.

Either I would name this `rawResponse()` or `rawResult()`; or change it to actually store the bytecode after creation and always return it..

Reply:
Isn't it just the content of returndata? Maybe `returndata()` would be more self-explanatory?

I had to look it up so here's some context for anyone else looking at it:
https://github.com/ethereum/solidity/blob/c7e474f243fa6bd6d6c172118b95d52113b5fbad/test/ExecutionFramework.cpp#L183

https://github.com/ethereum/solidity/blob/5bfd26287c6031f1564c5a8978d0e37b2f4427c2/test/evmc/evmc.h#L356-L367